packages/chat-stateful-client/src/iterators/createDecoratedIterator.ts

```diff
@@ -4,6 +4,7 @@
 import { ListPageSettings } from '@azure/communication-chat';
 import { ChatContext } from '../ChatContext';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { ChatErrorTargets } from '../ChatClientState';
 
 type IteratorCreatorFn<T, OptionsType, PageT = T[]> = (options?: OptionsType) => PagedAsyncIterableIterator<T, PageT>;
 
```

Comment:
Sry, this request is unrelated, but I just noticed in line 24/25, I named the iterator as threadsIterator, which I believe is not generic enough because it refers not only to thread now, could you please help me make a change - either iterator or innerIterator would be good for this case

Reply:
Will do at the top of the stack.